client/cmd/cliente2e/main.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2017 The etcd-operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"os"
+
+	"github.com/coreos/etcd-operator/client/experimentalclient"
+	"github.com/coreos/etcd-operator/pkg/spec"
+
+	"github.com/Sirupsen/logrus"
+	apierrors "k8s.io/client-go/pkg/api/errors"
+)
+
+func main() {
```

Comment:
as initial test, it is ok for me. But we need to separate these tests in the future.